go/dbinfo/dbinfo.go

```diff
@@ -47,14 +45,24 @@ func run(out io.Writer, cfg Config) error {
 	return err
 }
 
+type TableColumn struct {
+	Name       string `json:"name"`
+	Type       string `json:"type"`
+	KeyType    string `json:"keyType,omitempty"`
+	IsNullable bool   `json:"isNullable"`
```

Comment:
nitpick: could we slim up the json output? like removing `isNullable` for false values?